assets/src/edit-story/elements/text/edit.js

```diff
@@ -316,16 +326,22 @@ function TextEdit({
       onClick={onClick}
       data-testid="textEditor"
       backgroundColor={wrapperBackgroundColor}
+      textPadding={textProps.padding}
     >
       {editorContent && backgroundTextMode === BACKGROUND_TEXT_MODE.HIGHLIGHT && (
-        <TextBox ref={highlightRef} {...textProps}>
+        <TextBox ref={highlightRef} {...textProps} padding={null}>
```

Comment:
Can we unset the value in `textProps` instead? Or cloning `textProps` to a new `highlightTextPops` omitting the value?

Reply:
Done, it's cleaner now.